docs/integrations/enterprise-connectors/destination-hubspot.md

```diff
@@ -0,0 +1,55 @@
+---
+dockerRepository: airbyte/destination-salesforce
+---
+# HubSpot Destination
+
+## Overview
+
+This page guides you through the process of setting up the [HubSpot](https://www.hubspot.com/) destination connector.
+
+## Prerequisites
+
+- HubSpot Account
+
+## Setup guide
+
+### Step 1: Set up the HubSpot connector in Airbyte
+
+1. [Log into your Airbyte Cloud](https://cloud.airbyte.com/workspaces) account.
+2. Click Destination and then click + New destination.
+3. On the Set up the source page, select HubSpot from the Destination type dropdown.
```

Comment:
nit: this has not been a dropdown for a long time 😄 
<img width="1248" height="391" alt="image" src="https://github.com/user-attachments/assets/d04b3c13-5ea4-493e-a2ab-0776a235160b" />